### ISSUE TITLE
Crypton throws CSP JS Error in Firefox 40.0.3 (OS X) : Failed to load script (nsresult = 0x805e0006)

### ISSUE THREAD

**Issue by grempe** [SpiderOak/crypton#392]
In Firefox (40.0.3) I see the following issue on page load of the Items example (untried for others).

````
Error: Failed to load script (nsresult = 0x805e0006) crypton.js:6291
````

I think this compiled crypton.js is tied to this code:

https://github.com/SpiderOak/crypton/blob/4c764298df5eb89033806da960ac7dc23a036a55/client/src/vendor/isomer.js#L52

This seems to be related to Firefox handling of Content Security Policy for blob: type.

https://bugzilla.mozilla.org/show_bug.cgi?id=1037768

I have tried various incantations of including blob: in the CSP but was unable to clear this console error. I don't see a related error in Chrome or Safari on OS X.

**Comment by daviddahl** [comment]
Yeah, I think the solution is to create crypton-worker.js and not be so clever with the Blob-based worker, which is perhaps "too clever by half" :P
